src/controllers/generator.controller.ts

```diff
@@ -0,0 +1,67 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import { ProblemException } from '../exceptions/problem.exception';
+
+import { Controller } from '../interfaces';
+
+import { generatorTemplateParametersValidationMiddleware } from "../middlewares/generator-template-parameters-validation.middleware";
+import { ArchiverService } from '../services/archiver.service';
+import { GeneratorService } from '../services/generator.service';
+import { prepareParserConfig } from '../utils/parser';
+
+/**
+ * Controller which exposes the Generator functionality
+ */
+export class GeneratorController implements Controller {
```

Comment:
```suggestion
export class GenerateController implements Controller {
```

Also apply this name change globally to any other place.

Reply:
ok 👍🏼